azure-application-insights-spring-boot-starter/src/main/java/com/microsoft/applicationinsights/autoconfigure/ApplicationInsightsTelemetryAutoConfiguration.java

```diff
@@ -144,6 +144,19 @@ public TelemetryConfiguration telemetryConfiguration(TelemetryChannel telemetryC
         telemetryConfiguration.setChannel(telemetryChannel);
         initializeComponents(telemetryConfiguration);
         initializePerformanceCounterContainer();
+
+        ApplicationInsightsProperties.Exceptions exceptionsConfig = applicationInsightsProperties.getExceptions();
+        if (exceptionsConfig != null) {
+            Integer maxStackSize = exceptionsConfig.getMaxStackSize();
+            if (maxStackSize != null) {
```

Comment:
If this is null, shouldn't we have a default?

Reply:
Done